src/wagtailmedia/views/chooser.py

```diff
@@ -253,9 +244,7 @@ def chooser_upload(request, media_type):
     }
     return render_modal_workflow(
         request,
-        "wagtailmedia/chooser/chooser.html"
-        if WAGTAIL_VERSION >= (3, 0)
-        else "wagtailmedia/chooser/chooser-legacy.html",
```

Comment:
Now that we no longer need the legacy templates, the need to be removed. 

`wagtailmedia/chooser/chooser-legacy.html`
`wagtailmedia/chooser/legacy/`
`wagtailmedia/homepage/legacy_site_summary_media.html`
`wagtailmedia/media/legacy/`

Reply:
Removed legacy templates and references to those.